README.md

```diff
@@ -276,6 +276,8 @@ For long term caching use `filename: "[contenthash].css"`. Optionally add `[name
 
 If you'd like to extract the media queries from the extracted CSS (so mobile users don't need to load desktop specific CSS anymore) you should use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin). It will take over the filename option of the mini-css-extract-plugin and recognize its generated CSS chunks.
 
+Or you can use [media-query-splitting-plugin](https://github.com/mike-diamond/media-query-splitting-plugin), it will handle mobile, tablet and desktop styles splitting for each CSS chunk in your bundle without any configuration, just add it to the webpack plugins.
```

Comment:
Please make the description more generic e.g

```
- you should use the [media-query-plugin](https://github.com/SassNinja/media-query-plugin) ...
+ you should use one of the following plugins ...
```

and make a list for the plugins instead

```
... its generated CSS chunks.
 
- [Media Query Plugin](media-query-plugin)
- [Media Query Splitting Plugin](media-query-splitting-plugin)

[media-query-plugin]: https://github.com/SassNinja/media-query-plugin
[media-query-splitting-plugin]: https://github.com/mike-diamond/media-query-splitting-plugin
```

Reply:
Fixed